docs/Makefile.am

```diff
@@ -415,6 +415,7 @@ OMPI_MAN3 = \
         MPI_Status_set_cancelled.3 \
         MPI_Status_set_elements.3 \
         MPI_Status_set_elements_x.3 \
+        MPI_T.3 \
```

Comment:
Thanks for fixing the Makefiles.  Can you squash this commit into the other commit where you moved the files to different sections?  We don't need to have a commit on the PR and then a later commit on the same PR that fixes the previous commit.  Thanks!